src/test/java/com/google/edith/UserInsightsTest.java

```diff
@@ -106,15 +108,16 @@ public void aggregateUserData_withQuantityTwoOrLess_returnsCorrectJsonString() {
     datastore.put(newEntity); 
 
     Entity newEntity2 = new Entity(items.get(1));
-    setEntityProperties(newEntity2, "potato", 6, 2, "2020-06-30");
+
+    setEntityProperties(newEntity2, "corn", 6, 2, "2020-06-30");
     datastore.put(newEntity2); 
     
-    Map<String, String> map = new LinkedHashMap<String, String>();
-    map.put("2020-07-05", "17.0");
+    List<WeekInfo> expected = new ArrayList<WeekInfo>();
```

Comment:
Could we use ImmutableList here instead? It's best to prefer immutable objects wherever possible.


https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableList.html

Reply:
Oh sure!